modules/commons/src/main/resources/schemas/neurosciencegraph/commons/annotation/v0.2.0.json

```diff
@@ -0,0 +1,202 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.2.0",
+    {
+      "@base": "{{base}}/schemas/neurosciencegraph/commons/annotation/v0.2.0/shapes/"
+    }
+  ],
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v1.0.0",
+    "{{base}}/schemas/neurosciencegraph/commons/selectors/v0.2.0"
+  ],
+  "@type": "nxv:Schema",
+  "shapes": [
+    {
+      "@id": "AnnotationShape",
+      "@type": "sh:NodeShape",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "class": "oa:Annotation",
```

Comment:
should we replace this with a comment? Or add a comment to point out that we capture the oa:Annotation description in these shapes? Also, can "sh:class" be used outside property shapes?

Reply:
I did it to so that `oa:Annotation` is in `@type`for all instances using directly or indirectly `AnnotationShape`.